problems/[230817] 6. Zigzag Conversion.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * https://leetcode.com/problems/zigzag-conversion/
+ * @param {string} s
+ * @param {number} numRows
+ * @return {string}
+ */
+const convert = function (s, numRows) {
+  if (numRows === 1 || s.length < numRows) return s;
+
+  let count = 0;
+  let direction = false;
+  const result = [];
+
+  for (let i = 0; i < numRows; i += 1) {
+    result[i] = [];
+  }
```

Comment:
이 부분은 생성자 함수와 메서드를 사용해서 간편하게 작성이 가능할 것 같아요.
```suggestion
  const result = Array(numRows).fill([]);
```